### PR TITLE
Fixes issue where scrollbar is all the way on the right

### LINKED ISSUE #532: Remove scroll bar from right hand side and show it as part of the left most div container.
## Background
The [map](https://theshaft.info/explore) isn't scrollable but the left most div is. It's a little unintuitive for users.

Screen Shot 2023-01-22 at 10 57 50 AM

### PR DESCRIPTION
**Pull request by Etesam913**
### Pretty much solved the bug by setting a max-height property on the sidebar container.

Video of the bug fix:

https://user-images.githubusercontent.com/55665282/221428778-53db1e45-89ca-4f3b-944d-10437cda9e36.mp4

### CODE REVIEW

File: src/client/src/components/ExploreFilters/Filters.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ let FilterRow = styled.div`
 const Filters = styled.div`
 	display: flex;
 	flex-direction: row;
+	overflow-x: scroll;
```
Comment (Etesam913) [inline src/client/src/components/ExploreFilters/Filters.js:31]: Small QoL fix

File: src/client/src/components/ExploreSidebar.js [hunk 2]
```diff
@@ -1,60 +1,68 @@
...
-    height: 100%;
-    display: grid;
-    grid-column-gap: 5%;
-    grid-template-columns: 47.5% 47.5%;
-`
+  margin: 0 auto;
+  padding: 0 5%;
+  margin-top: 2vh;
+  overflow: scroll;
+  max-height: 70vh;
```
Comment (Etesam913) [inline src/client/src/components/ExploreSidebar.js:13]: This is very important in getting the correct scroll behavior

File: src/client/src/containers/Explore.js [hunk 3]
```diff
@@ -17,50 +17,50 @@ const ExploreContainer = styled.div`
...
   flex-direction: row;
-`
+`;
 
 const SideBar = styled.div`
   width: 100%;
-  padding: 0% 0% 0% 0%;
-  overflow-y: scroll; 
+  padding: 0 0 0 0;
+  overflow-x: hidden;
```
Comment (Etesam913) [inline src/client/src/containers/Explore.js:25]: Helped to fix a different overflow bug with the ad

**Comment by zhuci** [comment]
LGTM! thanks for the through explanations + video

**Pull request merged**